### PR TITLE
Add and update media sites in our collection

### PR DESCRIPTION
**Pull request by iMattPro**

### CODE REVIEW

File: collection/sites/twitter.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Twitter
```
Comment (rxu) [inline collection/sites/twitter.yml:1]: Din't work for me with `x.com` links on a clean 3.3.10 install. Worked after changing the name attribute, f.e. to `Twitter/X.com`.

Comment (rxu) [reply collection/sites/twitter.yml:1]: Well, I've made a fresh phpBB installation and then installed new ext code.
But ok, I'll double check this.

Comment (iMattPro) [reply collection/sites/twitter.yml:1]: Asking cuz it works fine for me as is, which is also how it is currently set up in the latest from s9e. Also the name field should not affect the embedding.

Comment (rxu) [reply collection/sites/twitter.yml:1]: Okay, rechecked and it worked this time. Probably there're some Interbet connection issues on my side.

**Review by rxu — APPROVED** [review]

**Pull request merged**